pyomo/gdp/plugins/chull.py

```diff
@@ -790,6 +775,9 @@ def _xform_constraint(self, obj, disjunct, infodict, var_substitute_map,
 
             if c.equality:
                 if NL:
+                    # ESJ TODO: This can't happen right? This is the only
+                    # obvious case where someone has messed up, but this has to
+                    # be nonconvex, right? Shouldn't we tell them?
                     newConsExpr = expr == c.lower*y
```

Comment:
@jsiirola, one question: Shouldn't we throw an error here since it is definitely non-convex? Or am I crazy?

Reply:
You are correct that it would be non-convex, but I think the Hull Reformulation described by doing the following is still a valid reformulation of the original problem. As such, `gdp.chull` has always been somewhat false advertising.

@bernalde You want to double-check this?